oap-server/oal-grammar/src/main/antlr4/org/apache/skywalking/oal/rt/grammar/OALParser.g4

```diff
@@ -87,7 +87,9 @@ functionName
 funcParamExpression
     : expression
     ;
-
+arrayFuncParamExpression
+    : LS_BRACKET ((funcParamExpression|literalExpression|attributeExpression) (COMMA (funcParamExpression|literalExpression|attributeExpression))*) RS_BRACKET
```

Comment:
The element of an array could be `funcParamExpression`? Could you demo how it would be?

Reply:
```
mq_transmission_latency = from(MessageQueueAccess.transmissionLatency).filter(operation == MessageQueueOperation.Consume).labeledLongAvg([topic,queue,queue,"1"]);

```

Like  this , we  could  use  `[a]` to represent an arg

Reply:
```
@Entrance
    public final void combine(@SourceFrom long value, @ConstOne long count, @Arg String[] labels)
```

Reply:
I  recked `funcParamExpression ` , you  are right  . I should  remove  `funcParamExpression `

Reply:
My question is about `funcParamExpression`, what you mentioned is `attributeExpression`(topic,queue,queue).